api/src/main/java/org/openmrs/BaseCustomizableData.java

```diff
@@ -24,9 +24,9 @@
  * @param <A> the type of attribute held
  * @since 1.9
  */
-public abstract class BaseCustomizableData<A extends Attribute> extends BaseChangeableOpenmrsData implements Customizable<A> {
+public interface Attribute<AT extends AttributeType, OT extends Customizable<?>> extends OpenmrsData, SingleCustomValue<AT>, Comparable<Attribute> {
```

Comment:
```suggestion
public abstract class BaseCustomizableData<A extends Attribute> extends BaseChangeableOpenmrsData implements Customizable<A> {
```

Reply:
We want to change the interface for [this class](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/attribute/Attribute.java), rather than changing `BaseCustomizableData`.